sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala

```diff
@@ -43,7 +44,7 @@ class V2SessionCatalog(catalog: SessionCatalog)
   extends TableCatalog with FunctionCatalog with SupportsNamespaces with SQLConfHelper {
   import V2SessionCatalog._
 
-  override val defaultNamespace: Array[String] = Array("default")
+  override val defaultNamespace: Array[String] = Array(SQLConf.get.defaultDatabase)
```

Comment:
`catalog.defaultDatabase`? It's already available in `SessionCatalog`

Reply:
catalog.defaultDatabase results the following compilation error, therefore I have not modified this part:

```
[ERROR] /Users/roczei/github/CDPD-40362/upstream/spark/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala:46: value defaultDatabase is not a member of org.apache.spark.sql.catalyst.catalog.SessionCatalog
[ERROR] one error found
```